docker/install_libs.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
```

Comment:
This should be a good comprise to install the libs without pollute the dockerfile